red/red.js

```diff
@@ -36,6 +38,7 @@ var RED = {
         userSettings.version = this.version();
         settings.init(userSettings);
         server.init(httpServer,settings);
+        memoryMetricsReporter.init(); //TODO pass in interval settings
```

Comment:
@hbeeken is adding a setting to enable metric reporting. This should tie to that.